apps/condo/k6/src/registerMetersReadings.test.ts

```diff
@@ -0,0 +1,106 @@
+import { faker } from '@faker-js/faker/locale/ru'
+import dayjs from 'dayjs'
+import { check } from 'k6'
+
+import { COLD_WATER_METER_RESOURCE_ID } from '@condo/domains/meter/constants/constants'
+import { FLAT_UNIT_TYPE } from '@condo/domains/property/constants/common'
+
+import {
+    createOrganization,
+    createProperty,
+    getOrganizationEmployeeId,
+    sendAuthorizedRequest,
+    setupCondoAuth,
+} from './utils'
+
+const DURATION = '60s'
+
+export const options = {
+    tags: { testid: 'registerMetersReadings', serverUrl: __ENV.BASE_URL },
+    scenarios: {
+        registerMetersReadings: {
+            exec: 'registerMetersReadings',
+            executor: 'constant-vus',
+            duration: DURATION,
+            vus: 1,
+        },
+    },
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        http_req_duration: ['med<5000'],
```

Comment:
Why 5000 is here? Seems like a lot, doesn't it?

Reply:
decrease to 1s